README.md

```diff
@@ -71,7 +71,12 @@ Redis | ✅ | ❌ | [swift-server/RediStack](https://github.com/swift-server/Red
 
 This is the current version of SwiftNIO and will be supported for the foreseeable future.
 
-The most recent versions of SwiftNIO support Swift 5.7 and newer. The minimum Swift version supported by SwiftNIO releases are detailed below:
+### Swift Versions
+
+The intention is to support the most recently released swift version (currently 5.10) and the last two minor releases before that.
```

Comment:
I think we should punch this up a bit more. It's not our intention, it's our commitment, unless we find ourselves in a place where this simply is not possible. Let's be a bit more forceful.

Reply:
fine with me, will add 👊